web/src/router/index.ts

```diff
@@ -18,26 +23,33 @@ const routes: RouteRecordRaw[] = [
       },
     ],
   },
-
   {
     path: '/404',
     name: '404',
     component: () => import('@/views/exception/404/index.vue'),
   },
-
   {
     path: '/500',
     name: '500',
     component: () => import('@/views/exception/500/index.vue'),
   },
-
   {
     path: '/:pathMatch(.*)*',
     name: 'notFound',
     redirect: '/404',
   },
 ]
 
+// !!!
+// https://router.vuejs.org/guide/essentials/history-mode.html
+// createWebHashHistory
+// It uses a hash character (#) before the actual URL that is internally passed.
+// Because this section of the URL is never sent to the server,
+// it doesn't require any special treatment on the server level.
+// It does however have a bad impact in SEO. If that's a concern for you, use the HTML5 history mode.
+
+// this is crazy, router in frontend is a nightmare
+
 export const router = createRouter({
   history: createWebHashHistory(),
   routes,
```

Comment:
:robot: OpenAI

This patch adds some comments to the file. The comments explain that the `createWebHashHistory()` function is used for the router, and that it uses a hash character before the actual URL. The comments also mention that this has a bad impact on SEO, and suggest using HTML5 history mode if that's a concern.

The comment "this is crazy, router in frontend is a nightmare" seems like a personal opinion and should be removed. 

Overall, the comments are helpful and provide useful information about the router implementation. However, the comment about the router being a nightmare may not be appropriate for a code review.

<!-- This is an auto-generated comment by OpenAI -->